packages/web-component-tester/custom_typings/resolve-from.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'resolve-from' {
```

Comment:
We already use https://www.npmjs.com/package/resolve which has this functionality.